capsules/src/acifc.rs

```diff
@@ -0,0 +1,61 @@
+//! Provides userspace access to the ACIFC interface.
+//!
+//! ## Instantiation
+//! -----
+//!
+//! ```rust
+//! let acifc = static_init!(
+//! capsules::acifc::Acifc<'static, sam4l::acifc::Acifc>,
+//! capsules::acifc::Acifc::new(&mut sam4l::acifc::ACIFC);
+//! ```
+//!
+//! Author: Danilo Verhaert <verhaert@cs.stanford.edu>
+
+/// Syscall driver number.
+pub const DRIVER_NUM: usize = 0x07;
```

Comment:
If we're adding a cross-platform system call driver, should update `doc/syscalls/README.md` and document the interface in `doc/syscalls/00007_analog_compartor.md`.

Reply:
Done!